### PR TITLE
logic for SMS send window

### PR DESCRIPTION
**Pull request by acn-sbuad**
## Related Issue(s)
- #538 

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)

### CODE REVIEW

File: test/Altinn.Notifications.Tests/Notifications/TestingControllers/TriggerControllerTests.cs [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        {
+            // Arrange
+            var dateTime = new DateTime(2022, 1, 1, 10, 0, 0, DateTimeKind.Utc);
+
+            // Act
+            var result = dateTime.IsWithinBusinessHours();
+
+            // Assert
+            Assert.True(result);
+        }
```
Comment (SandGrainOne) [inline test/Altinn.Notifications.Tests/Notifications/TestingControllers/TriggerControllerTests.cs:60]: This test doesn't test the controller

File: src/Altinn.Notifications/Configuration/GeneralSettings.cs [hunk 2]
```diff
@@ -24,4 +24,14 @@ public class GeneralSettings
     /// Default sender of email notifications
     /// </summary>
     public string DefaultEmailFromAddress { get; set; } = "noreply@altinn.no";
+
+    /// <summary>
+    /// Start hour of the SMS send window
+    /// </summary>
+    public static int SmsSendWindowStartHour { get; set; } = 9;
```
Comment (SandGrainOne) [inline src/Altinn.Notifications/Configuration/GeneralSettings.cs:31]: Honestly not sure I like these new properties being static. I assume it's still possible to set the value through appsettings and other configuration sources?

File: test/Altinn.Notifications.Tests/Notifications/TestingControllers/TriggerControllerTests.cs (hunk 1 above)
Comment (acn-sbuad) [reply test/Altinn.Notifications.Tests/Notifications/TestingControllers/TriggerControllerTests.cs:60]: I blame copilot for an unrelated test. and myself for not checking better. Fixing..

File: src/Altinn.Notifications/Configuration/GeneralSettings.cs (hunk 2 above)
Comment (acn-sbuad) [reply src/Altinn.Notifications/Configuration/GeneralSettings.cs]: created a seperate service for schedule to limit dependencies for controller. Config values moved to NotificationOrderConfig. We could consider renaming the config section to NotificationConfig

**Review by SandGrainOne — APPROVED** [review]

File: src/Altinn.Notifications/Configuration/GeneralSettings.cs (hunk 2 above)
Comment (SandGrainOne) [reply src/Altinn.Notifications/Configuration/GeneralSettings.cs]: I changed the name.

**Pull request merged**